Cargo.toml

```diff
@@ -48,6 +48,7 @@ rayon = "1.8.0"
 
 [dev-dependencies]
 assert_cmd = "2.0"
+insta = { version = "1.36.1", features = ["yaml"] }
```

Comment:
Adding the following at the bottom of this file might be benefitial according to their docs

```toml
# See https://insta.rs/docs/quickstart/#optional-faster-runs
[profile.dev.package]
insta.opt-level = 3
similar.opt-level = 3
```

Reply:
This PR is a bit outdated at this point, and the `Cargo.toml` already has a broader version of those directives already: `profile.dev.package."*"` is `opt-level = 3`